platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Hash.java

```diff
@@ -156,6 +135,13 @@ public byte[] getValue() {
         return value;
     }
 
+    /**
+     * @return a copy of the hash value as a byte array
+     */
+    public byte[] copyToByteArray() {
+        return Bytes.EMPTY.toByteArray();
+    }
```

Comment:
Is this a place holder? It seems like this is always just returning an empty array.

```suggestion
    @NonNull
    public byte[] copyToByteArray() {
        return Bytes.EMPTY.toByteArray();
    }
```

Reply:
Yeah, I believe we want to return `this.bytes.toByteArray()` here.

Reply:
this is a big mistake, no idea how it happened. fixed